source/docs/software/wpilib-tools/robot-simulation/introduction.rst

```diff
@@ -0,0 +1,45 @@
+Introduction to Robot Simulation
+================================
+
+Often a team may want to test their code without having an actual robot available. WPILib provides teams with the ability to simulate the robot's Hardware Abstraction Layer (HAL) using simple gradle commands.
```

Comment:
Can we simply say "simulate the robot's hardware" rather than mentioning the HAL?